CalculatorIdo/Script.js

```diff
@@ -0,0 +1,149 @@
+window.onload = () => {clearCalculator();}
+
+var currAns = '';
+var isResultShown = false;
+var isbracketOpen = false;
+
+clearCalculator = () => {document.calcForm.input.value = '';}
+
+del = () => {
```

Comment:
Try to minimize writing global functions, it makes testing much harder. for example if you want to develop an interface for your calculator and change the implementation it won't be possible